goldendict.pro

```diff
@@ -642,6 +642,7 @@ TRANSLATIONS += locale/ru_RU.ts \
     locale/fi_FI.ts \
     locale/jb_JB.ts \
     locale/hi_IN.ts \
+    locale/ca_ES.ts \
```

Comment:
Move this line just under the line `locale/es_ES.ts \`, where it belongs.

Reply:
I think the changed line still should be moved under the Spanish translation because of its closeness. Everything else looks good to me and works as expected.

Reply:
Done! :)